cli.md

```diff
@@ -53,9 +53,15 @@ If `--name` is not specified, CLI will look for an environment variable, e.g. `e
 * `t2 run <file>` copy the file and its dependencies into Tessel's RAM & run immediately. Use this during development of your device application.
   * `[--lan]` deploy over LAN connection
   * `[--usb]` deploy over USB connection
+  * `[--slim]` true by default, copy only files needed by the program to run
+  * `[--slimPath]` "__tessel_program__.js" by default, name of the temporary slim bundle to copy to Tessel
```

Comment:
I'm okay with `--slim` staying. Could be good if the default (between `slim` and `full`) can be specified via config or env-variable. Then you can always override it with the flag.